client/blocks/reader-subscription-list-item/index.jsx

```diff
@@ -63,6 +65,23 @@ function ReaderSubscriptionListItem( {
 		return <ReaderSubscriptionListItemPlaceholder />;
 	}
 
+	function recordEvent( name ) {
```

Comment:
yeah, I don't love this. the "right" way to handle this is to bake the ID(s) we need into data attributes and then look up the right data from a bound event handler. That means turning this into a full Component...

As a leaf though, it's less of an issue. 

I _am_ seeing it rerender quite a bit more than I would expect though?